src/pages/settings/Subscription/CardSection/utils.ts

```diff
@@ -32,6 +40,13 @@ function getBillingStatus(translate: LocaleContextProps['translate'], accountDat
 
     const isCurrentCardExpired = DateUtils.isCardExpired(accountData?.cardMonth ?? 0, accountData?.cardYear ?? 0);
 
+    const purchaseAmount = purchase?.message.billableAmount;
+    const purchaseCurrency = purchase?.currency;
+    const purchaseDate = purchase?.created;
+    const isBillingFailed = purchase?.message.billingType === 'failed_2018';
```

Comment:
Let's pull `failed_2018` out into a constant, `BILLING.TYPE_FAILED_2018` to keep things consistent with our backend repos